docs/management_api.md

```diff
@@ -5,7 +5,7 @@ sort_rank: 7
 
 # Management API
 
-Prometheus provides a set of management API to ease automation and integrations.
+Prometheus provides a management API to facilitate automation and integration.
```

Comment:
"set of management APIs" is correct

Reply:
Changed.  `API` wasn't plural as I found it.  I've made it so.